Makefile

```diff
@@ -39,6 +39,18 @@ endif
 ARCHIVE_NAME=dist/fasttrackml_$(shell go env GOOS | sed s/darwin/macos/)_$(shell go env GOARCH | sed s/amd64/x86_64/).$(ARCHIVE_EXT)
 ARCHIVE_FILES=$(APP) LICENSE README.md
 
+# Create the platform name 
+ifeq ($(shell go env GOOS)_$(shell go env GOARCH),darwin_x86_64)
+    PLATFORM_NAME := macosx_10_13_x86_64
+else ifeq ($(shell go env GOOS)_$(shell go env GOARCH),darwin_arm64)
+    PLATFORM_NAME := macosx_11_0_arm64
+else ifeq ($(shell go env GOOS)_$(shell go env GOARCH),linux_x86_64)
+    PLATFORM_NAME := manylinux1_x86_64
+else ifeq ($(shell go env GOOS)_$(shell go env GOARCH),linux_arm64)
+    PLATFORM_NAME := manylinux1_aarch64
+else ifeq ($(shell go env GOOS)_$(shell go env GOARCH),windows_x86_64)
+    PLATFORM_NAME := win_amd64
+endif
```

Comment:
do this within `setup.py` to return the proper `plat` in `bdist_wheel.get_tag()`